### ISSUE TITLE
setAnalyticsEnabled is removed in iOS but not android

### ISSUE THREAD

**Issue by sdenaci** [urbanairship/react-native-module#377]
❗For how-to inquiries involving Airship functionality or use cases, please
contact (support)[https://support.airship.com/].

# Preliminary Info

### What Airship dependencies are you using?
    "urbanairship-react-native": "13.1.0",

### What are the versions of any relevant development tools you are using?
xcode 13.1

# Report

### What unexpected behavior are you seeing?
when I try to call setAnalyticsEnabled on iOS I am getting a crash `TypeError: UrbanAirshipModule.setAnalyticsEnabled is not a function. (In 'UrbanAirshipModule.setAnalyticsEnabled(enabled)', 'UrbanAirshipModule.setAnalyticsEnabled' is undefined)` I am not getting it on Android. it would seem that it was removed on iOS in October https://github.com/urbanairship/react-native-module/commit/9d811ca97a0b374b013ca1cf2939f3164724b479, but never replaced...?

### What is the expected behavior?
To be able to call setAnalyticsEnabled without a crash

### What are the steps to reproduce the unexpected behavior?
call setAnalyticsEnabled(true) on iOs using urban-airship-react-native 13.1.0

**Comment by Ulrico972** [comment]
Hello,
I think we forgot to clean this right. We'll do it properly in a next version, sorry for the inconvenience.

Actually you shouldn't use the `setAnalyticsEnabled()` method anymore.
Instead use the `enabledFeature()` and `disableFeature()` methods. (https://docs.airship.com/platform/react-native/data-collection/#enabling-features-at-runtime)

**Comment by rlepinski** [comment]
Yeah it was replaced with enableFeatures but not fully removed. we will do a patch to restore it since it was not fully removed.
